setup.py

```diff
@@ -29,11 +29,22 @@
 
 here = path.abspath(path.dirname(__file__))
 
+with open(path.join(here, 'README.md'), encoding='utf-8') as f:
+    long_description = f.read()
+
+with open(path.join(here, 'requirements.txt'), encoding='utf-8') as f:
+    install_requires = []
+    for line in f.readlines():
+        if line and line[0].isalpha():
+            install_requires.append(line.strip())
+
 # The version is set at build time
 setup(
```

Comment:
missing `install_requires` argument?

Reply:
Good catch. I cannot believe I overlooked that!